CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 ## next
 
-## 1.1.5
+## 2.0.0
 
 *Bug Fixes*
+- Always set a deployment_id, even if the current_sha isn't set [#730](https://github.com/Shopify/krane/pull/730)
 - YAML string scalars with scientific/e-notation numeric format not properly quoted. [#740](https://github.com/Shopify/krane/pull/740)
 
+*Other*
```

Comment:
```suggestion
*Breaking Changes*
```

Also, I would have it as the first header. I would argue #730 is also a breaking change